crates/fluvio-controlplane-metadata/src/mirror/status.rs

```diff
@@ -36,9 +68,11 @@ pub enum MirrorPairStatus {
     #[fluvio(tag = 1)]
     Succesful,
     #[fluvio(tag = 2)]
-    Failed,
+    Failed(String),
```

Comment:
since this is protocol change, should bump up fluvio version

Reply:
Sure, minor right? done